UnitsNet/UnitConverter.cs

```diff
@@ -44,6 +45,14 @@ public static class UnitConverter
         private static readonly string UnitTypeNamespace = typeof(LengthUnit).Namespace;
         private static readonly Assembly UnitsNetAssembly = typeof(Length).GetAssembly();
 
+        private static readonly Type[] QuantityTypes = UnitsNetAssembly.GetTypes()
```

Comment:
These include a lot more types than just the quantity structs and the unit enums.
I'm adding a fix.